### DISCUSSION TITLE
ok.ru channel

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#222] [Q&A] [answered]
it allowed or not .. to add those channel  in list  ?

the only way  to get them working,  is by using  @keystroke3  proxy 

there 100s of live channels

**Comment by keystroke3** [comment, marked as answer]
I think it should be OK. There are some channels that are only available on youtube and need the proxy. But maybe list the channels so the maintainers can look at them.

**Reply by Sphinxroot** [reply]
https://ok.ru/live/search/?st.mode=Movie&st.query=tv&st.grmode=Groups as exemple of a small list   if you change the query  u get other stuff 


Kodi
#EXTINF:-1,MAFI2A MUSIC TV
https://linkastream.co/headless?url=https://ok.ru/live/2246200336020|User-Agent=Mozilla/5.0 (Linux; Android 10) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.101 Mobile Safari/537.36


Other Player that support it
#EXTINF:-1 ,MAFI2A MUSIC TV
#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Linux; Android 10) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.101 Mobile Safari/537.36
https://linkastream.co/headless?url=https://ok.ru/live/2246200336020
